packages/paste-core/components/progress-steps/stories/vertical.stories.tsx

```diff
@@ -102,3 +110,66 @@ export const Internationalized: React.FC = () => {
     </ProgressSteps>
   );
 };
+
+export const WithContent: React.FC = () => {
+  return (
+    <ProgressSteps orientation="vertical">
+      <ProgressStepComplete as="div">
+        Data warehouse connected
+        <ProgressStepContent>
+          <Box paddingX="space10">
+            <Box display="flex" columnGap="space50" paddingY="space40">
+              <Avatar size="sizeIcon90" name="Twilio Paste" variant="entity" icon={BusinessIcon} />
+              <Box>
+                <Heading as="h2" variant="heading50">
+                  Snowflake
+                </Heading>
+                <Box display="flex" flexDirection="column">
+                  <Paragraph>Account: accountname</Paragraph>
+                  <Paragraph>Database: snowflakedatabasename</Paragraph>
+                  <Paragraph>Warehouse: snowflakewarehousename</Paragraph>
+                  <Paragraph>User: bsmith</Paragraph>
+                  <Paragraph marginBottom="space0">Password: *****</Paragraph>
```

Comment:
suggestion(non-blocking): Use Description List instead of Paragraphs, and just show Account, Database, and Warehouse